### PR TITLE
Add ESLint linting and other refactoring

### PR DESCRIPTION
**Pull request by Dyskal**
Finally push the linting config.
Don't hesitate to tell me if you like it, if you want to add or remove some rules...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "@fortawesome/free-solid-svg-icons": "^6.4.2",
     "@fortawesome/vue-fontawesome": "^3.0.3",
     "@tailwindcss/forms": "^0.5.6",
+    "@types/markdown-it": "^13.0.5",
```
Comment (Dyskal) [inline package.json:18]: @types packages belong to dev dependencies

Comment (Dyskal) [reply package.json:18]: actually just remove this line it also exists below

File: src/components/Hero.vue [hunk 2]
```diff
@@ -1,22 +1,68 @@
...
+            stroke-linejoin="round"
+          />
+        </svg>
+      </div>
+    </div>
+  </div>
 </template>
 
-<style scoped>
+<style scope>
```
Comment (Dyskal) [inline src/components/Hero.vue:51]: @ShiroUsagi-san should this be scoped or not ? if not please remove the invalid attribute

Comment (ShiroUsagi-san) [reply src/components/Hero.vue]: solve in my last commit (7d47453)

**Review by ShiroUsagi-san — APPROVED** [review]

**Pull request merged**